packages/codemod/transformations/core/v2-to-v3/dialog-content-container-component-migration.ts

```diff
@@ -22,7 +23,7 @@ function transform({ j, root }: TransformationContext) {
     updatePropValues(j, elementPath, "size", {
       "DialogContentContainer.sizes.MEDIUM": {
         value: "DialogContentContainer.sizes.SMALL",
-        type: j.memberExpression
+        type: MemberExpression
```

Comment:
same, from j, just to not import more stuff in the transformations

I wonder if we should do it also for all the utils, I didn't do it initially because I didn't had `j` in all the utils (didn't needed it, like in the getCoreImports)